qiita_pet/handlers/api_proxy/tests/test_prep_template.py

```diff
@@ -35,5 +119,184 @@ def test_prep_template_get_req_no_access(self):
                'message': 'User does not have access to study'}
         self.assertEqual(obs, exp)
 
+    def test_prep_template_post_req(self):
```

Comment:
It doesn't look like there is a test in which prep_template_post_req works correctly (i.e. adds a prep template to the system).